modules/tinymce/src/core/test/ts/browser/init/InitIframeContentAriaLabelTest.ts

```diff
@@ -0,0 +1,30 @@
+import { describe, it } from '@ephox/bedrock-client';
+import { McEditor } from '@ephox/mcagar';
+import { Attribute, SugarElement } from '@ephox/sugar';
+import { assert } from 'chai';
+
+import Editor from 'tinymce/core/api/Editor';
+
+describe('browser.tinymce.core.init.InitIframeContentAriaLabelTest', () => {
```

Comment:
This test name and file needs updating as it's still using the old incorrect setting name.

- [ ] Fix incorrect test and file name